data_prep/project_context/context_introspector.py

```diff
@@ -283,46 +69,35 @@ def _get_files_to_include(self) -> list[str]:
     for current_type in types:
       info_list = introspector.query_introspector_type_info(
           self._benchmark.project, current_type)
-      print("INFO LIST: {}".format(info_list))
       if not info_list:
+        logging.debug('Could not retrieve info for type: %s', current_type)
```

Comment:
Could you please also log the project name?
This will help identify which project to debug :)

Also, `logging.warning` is more suitable here because this is unexpected and may indicate an error elsewhere.

Reply:
ACK done